app/views/books/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= @book.title %>
+
+<%= @book.reviews.each do |review|%>
+  <%= review.user.name %>
+  <%= review.body %>
+  <%= review.rating %>
+<% end %>
+
+<h2>Average Rating<h2>
+<%= Review.average_rating(@book.id)%>
```

Comment:
Another reason to write these method on Book is because I'm calling them from a book show. I don't want to have to call to another record's class methods in a view.